activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -11,9 +11,12 @@
 require "active_record/connection_adapters/sqlite3/schema_dumper"
 require "active_record/connection_adapters/sqlite3/schema_statements"
 
-gem "sqlite3", ">= 2.0"
+gem "sqlite3", ">= 2.1.0.rc3"
```

Comment:
What's the timeline for the final? Because I don't want to require an RC in a published release of Rails (well probably fine in a beta, but not a final)

Reply:
ASAP

Reply:
That's not a timeline. I would have done it today but @tenderlove raised a concern about using `close_v2` that I'd like to resolve before cutting the final.

Reply:
KK. Fine by me then. I'll merge on green.

Reply:
Ah, just worked it out with him. I'll cut a final release shortly, then I'll update this PR.

Reply:
Updated to `>= 2.1`.